server/crawler/crawler.go

```diff
@@ -491,29 +536,6 @@ func getRepositoryOwner(repository *github.Repository) (string, error) {
 
 }
 
-// completeRepository receives a probably uncompleted Repository object(for instance when it's returned by the Search endpoint)
-// and it will execute a request to GitHub for complete the data.
-// right now the only condition for completing it, is when the StargazersCount is nil because we need this data,
-// but we could add more conditions in the future
-func completeRepository(ctx context.Context, client *github.Client, currentRepository *github.Repository) (*github.Repository, error) {
```

Comment:
removing this method here because Im centralizing all calls in the `extractKurtosisPackageContent` method because it's used in both endpoints